include/freerdp/settings.h

```diff
@@ -403,6 +487,19 @@ extern "C"
 	FREERDP_API BOOL freerdp_settings_set_uint32(rdpSettings* settings,
 	                                             FreeRDP_Settings_Keys_UInt32 id, UINT32 param);
 
+	/** \brief Sets some UINT32 settings values.
+	 *
+	 *  \param settings A pointer to the settings to alter, must not be NULL.
+	 *  \param values The key/values to set
+	 *  \param n The number of key/value to set.
+	 *  \since version 3.11.0
+	 *
+	 *  \return \b TRUE for success, \b FALSE for failure
+	 */
+	FREERDP_API BOOL freerdp_settings_set_uint32s(rdpSettings* settings,
```

Comment:
maybe a good idea to make the name easier to distinguish from the single setters, name it `freerdp_settings_set_bulk(settings, values, count)`?